modules/bot/src/e2e.ts

```diff
@@ -0,0 +1,171 @@
+import { abrv, ChannelSigner, ColorfulLogger, getRandomPrivateKey } from "@connext/utils";
+import { constants, providers, utils, Wallet } from "ethers";
+import { Argv } from "yargs";
+
+import { env } from "./env";
+import { connect } from "@connext/client";
+import { getMemoryStore } from "@connext/store";
+import { Agent } from "./agents/agent";
+import { EventNames } from "@connext/types";
+
+const { AddressZero, Two } = constants;
+const { formatEther, parseEther } = utils;
+
+export const command = {
+  command: "e2e",
+  describe: "Run a quick e2e test",
+  builder: (yargs: Argv) => {
+    return yargs
+      .option("log-level", {
+        description: "0 = print no logs, 5 = print all logs",
+        type: "number",
+        default: 1,
+      })
+      .option("token-address", {
+        describe: "Asset id for payments",
+        type: "string",
+        default: AddressZero,
+      })
+      .option("chain-id", {
+        describe: "Chain ID to test",
+        type: "number",
+        default: 1337,
+      })
+      .option("funder-mnemonic", {
```

Comment:
We need to make sure to correctly pass in funder-mnemonic because it will have to be the node's mnemonic, not sugar daddy. Perhaps we should even just use whatever's in the env by default?